### DISCUSSION TITLE
Are operators published to operatorhub.io free to use?

### DISCUSSION THREAD

**Discussion by tisonkun** [k8s-operatorhub/community-operators#1794] [Q&A]
From the whole website, I don't find terms of service about this condition.

Said a company intends to use operatorhub.io as a free file server to share operators among its internal and client's usage, then its developer team submits the operator to operatorhub.io.

I'm a random free user who found the operator. The artifact submitted has no associated LICENSE or COPYING file. Am I safe to use it in any scenario I want to use?

**Comment by dmesser** [comment]
@tisonkun Licensing and usage permissions are always subject to the publisher. That said we envisioned operatorhub.io to be a place for the open source community to share their work - so naturally free-to-user is the targeted and most common scenario.
